build/pipelines/azure-pipelines.ci-internal.yaml

```diff
@@ -20,16 +20,24 @@ jobs:
 - template: ./templates/build-app-internal.yaml
   parameters:
     platform: x86
+    condition: not(eq(variables['Build.Reason'], 'PullRequest'))
```

Comment:
We should also run the ci internal pipeline for pull request reason. The condition in build-app-internal.yaml will use the non-release version for build and pass the UI test as well.